pkgdb/include/flox/core/util.hh

```diff
@@ -221,6 +222,12 @@ struct adl_serializer<nix::FlakeRef>
 
 }  // namespace nlohmann
 
+/* -------------------------------------------------------------------------- */
+
+/** @brief Detect if two vectors of strings are equal. */
+[[nodiscard]] bool
+operator==( const std::vector<std::string> & lhs,
+            const std::vector<std::string> & rhs );
```

Comment:
I personally don't care, but I know Alex prefers two blank lines before the closing `/* ----...`. Feel free to make a formatting pass as a followup.